python/ray/tests/test_output.py

```diff
@@ -15,6 +15,16 @@
 )
 
 
+def test_ray_init_no_redis_logs():
```

Comment:
I think this test actually doesn't run with redis.

Can you provide ray_start_regular_with_external_redis here?